doc/Programmer/pluginexample/CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ set( OpendTect_DIR "" CACHE PATH "OpendTect directory" )
 option( OD_BUILD_LOCAL "Build in local directory" ON )
 
 #Find OpendTect
-list(APPEND CMAKE_MODULE_PATH "${OpendTect_DIR}/CMakeModules")
+list(APPEND CMAKE_MODULE_PATH "${OpendTect_DIR}/${MISC_INSTALL_PREFIX}/CMakeModules")
```

Comment:
For OpendTect_DIR:
Do not make use of MISC_INSTALL_PREFIX, but check if the directory exists
For OD_BINARY_BASEDIR:
Use MISC_INSTALL_PREFIX, if different from OpendTect_DIR check also if directory exists:

if( ( NOT "${OD_BINARY_BASEDIR}/${MISC_INSTALL_PREFIX}" STREQUAL "${OpendTect_DIR}"  ) AND
     IS_DIRECTORY "${OD_BINARY_BASEDIR}/${MISC_INSTALL_PREFIX}/CMakeModules"  )
    list( APPEND ... )
endif()